.github/pr-commenter.yml

```diff
@@ -0,0 +1,67 @@
+comment:
+  on-update: edit
```

Comment:
If somebody pushes more commits that change more files, `on-update: edit` will update the existing comment made by this action. There is also `on-update: recreate` which would delete the old comment and create a new one, thus triggering a new notification. Not sure which one would be better for us.

Reply:
Do people get notified by an edit? If yes, then edit is probably fine. If no, recreate is probably better? 🤷🏻

Reply:
No, edits don't trigger notifications. I'll change it to `recreate` then 👍